WordPress/WordPressTest/ViewRelated/Tools/Time Zone/TimeZoneFormatterTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import WordPress
+@testable import WordPressKit
+
+class TimeZoneFormatterTests: XCTestCase {
+
+    // 2001-01-01 00:00:00 UTC
+    let testWinterDate = Date(timeIntervalSinceReferenceDate: 0)
```

Comment:
Calling these winter and summer might not make immediate sense to someone in the southern hemisphere – let's name them based on when in the year they fall instead.